.github/workflows/ci-build.yml

```diff
@@ -48,8 +48,15 @@ jobs:
     - name: Build
       run: dotnet build --no-restore -c ${{ env.Configuration }} /p:Version=${{ steps.gitversion.outputs.FullSemVer }}
 
-    - name: Run integration tests
-      run: dotnet test --no-build --verbosity normal -c ${{ env.Configuration }}
+    - name: Run tests
+      run: dotnet test --no-build --verbosity normal -c ${{ env.Configuration }} -p:CollectCoverage=true -p:CoverletOutputFormat=opencover -p:CoverletOutput=out/.coverage/
+
+    - name: Upload coverage reports to Codecov
+      uses: codecov/codecov-action@v3
```

Comment:
codecov v4 requires to pass `token` arg - breaking change, see https://github.com/destructurama/attributed/blob/master/.github/workflows/test.yml#L62-L67